### PR TITLE
Pat/nft gallery

### PR DESCRIPTION
**Pull request by nistadev**
NFT Gallery implementation for Yoroi Mobile

### CODE REVIEW

File: src/yoroi-wallets/types/types.ts [hunk 1]
```diff
@@ -57,3 +57,19 @@ export type YoroiAmount = {
 export type YoroiMetadata = {
   [label: string]: string
 }
+
+export type YoroiNFT = {
```
Comment (wolverineks) [inline src/yoroi-wallets/types/types.ts:61]: is this type defined in a library shared by mobile and extension?

File: src/components/ImageGallery/ImageGallery.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
+import React from 'react'
```
Comment (wolverineks) [inline src/components/ImageGallery/ImageGallery.tsx]: keep this colocated in /src/Nfts

File: src/Nfts/Nfts.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+  isDelegating: false,
+  isFetching: false,
+  lastFetchingError: '',
+  poolOperator: null,
+  totalDelegated: new BigNumber(0),
+  value: new BigNumber(0),
+}
+
+storiesOf('Nfts', module)
+  .add('not delegating', () => {
```
Comment (wolverineks) [inline src/Nfts/Nfts.stories.tsx]: copypasta?

File: src/yoroi-wallets/types/types.ts (hunk 1 above)
Comment (nistadev) [reply src/yoroi-wallets/types/types.ts:61]: I didn't found it, maybe we should include it in the `yoroi-lib` package

File: src/Nfts/Nfts.stories.tsx (hunk 3 above)
Comment (nistadev) [reply src/Nfts/Nfts.stories.tsx]: I will look into the storybook after, since I'll need a bit of help using it!

**Comment by michalszorad** [comment]
@stackchain @wolverineks should we close this PR in favor of https://github.com/Emurgo/yoroi-mobile/pull/2282 ? Please note the new PR is not ready yet

**Comment by stackchain** [comment]
closed in favor of #2282

**Pull request closed without merging**